CMakeLists.txt

```diff
@@ -8,7 +8,7 @@ set(EMAIL "mvourlakos@gmail.com")
 
 set(QT_MIN_VERSION "5.15.0")
 set(KF5_MIN_VERSION "5.81.0")
-set(KDECORATION2_MIN_VERSION "5.24.0")
+set(KDECORATION2_MIN_VERSION "5.26.90")
```

Comment:
i think it's ok to use `5.24.0`  and use `QString windowClass() const;` instead of `QString windowClass() const **override**;`, and then it can support both in 5.24 and 5.26.